src/tutorials/develop-and-deploy-a-flask-and-reactjs-app-in-okteto.mdx

```diff
@@ -0,0 +1,834 @@
+
+In this article, you'll be building a TODO list application directly in Okteto using the [Okteto Development Environment](https://okteto.com/docs/reference/development-environment/) with the Flask framework and ReactJS library.
+
+Remote development with Okteto doesn't require an active docker session. Instead, we provide you a persistent volume, instant file synchronization, and an accessible public endpoint to view your application changes as you develop your application locally.
+
+**[ReactJS](https://reactjs.org/)** is an open-source front-end JavaScript library used to build user interfaces.
+
+**[Okteto](https://okteto.com/)** is an open-source tool that runs locally to synchronize application code changes to a running pod in a local or remote Kubernetes cluster. With Okteto you're able to see a deployment version of your app as you're developing it.
+
+Instead of working locally, you'll use Okteto's development environment. Although your application can be built locally in your machine, using Okteto's development environment enables you to develop your application in a production ready environment.
+
+## Prerequisites
+
+In order to follow this article, the following is required:
+
+-   Basic understanding of Python and the [Flask](https://flask.palletsprojects.com/en/2.0.x/) framework
+-   Basic understanding of [ReactJS](https://reactjs.org/)
+-   Basic understanding of [Docker](https://docker.com/)
+-   [NPM](https://npmjs.org/) installed
+
+## Build Flask API
+
+The first step is to create a project directory to house our todo list application:
+
+```console
+$ mkdir flask-react-app
+$ cd flask-react-app
+```
+
+Next, create the folder for the API application and create a route file:
+
+```console
+$ mkdir api && cd api
+$ touch todo_list_api.py
+```
+
+Next, create a `requirements.txt` file where your application dependencies will be listed. In this case, you only have one, which is `Flask`:
+
+```txt
+Flask==1.1.2
+Flask-SQLAlchemy==2.5.0
+```
+
+[Flask-SQLAlchemy](https://flask-sqlalchemy.palletsprojects.com/en/2.x/) will be used to handle the database in which the todo data will be stored.
+
+Next, you're going to create a `Dockerfile`. This Dockerfile will contain the instructions for building your application's image:
+
+```console
+$ touch Dockerfile
+```
+
+Add the following to it:
+
+```Dockerfile
+FROM python:3.9
+
+WORKDIR /api
+
+ENV FLASK_ENV=development
+
+COPY ./requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD [ "python", "todo_list_api.py" ]
+```
+
+Your file structure should now look like this:
+
+```
+├── flask-react-app
+│   ├── api
+│   │   ├── todo_list_api.py
+│   |   ├── Dockerfile
+│   |   ├── requirements.txt		
+
+```
+
+## Scaffolding the frontend application
+
+For the frontend part of this application, you'll be using [Create-React-App](https://reactjs.org/docs/create-a-new-react-app.html/) to scaffold a new application.
```

Comment:
Since `create-react-app` is no longer the recommended way to create a new app and it's not even in the official docs linked here, could we consider using another approach in this tutorial?
The recommended way is to use a framework, but they also mention Vite which I think is the simplest way to be used in this tutorial.